.github/workflows/system-tests.yml

```diff
@@ -33,7 +49,7 @@ jobs:
 
       - name: Compress artifact
         if: ${{ always() }}
-        run: tar -czvf artifact.tar.gz $(ls | grep logs)
+        run: tar -czvf artifact.tar.gz $(ls | grep logs) || true
```

Comment:
Needed as the library injection tests don't create or use the logs dir (this should probably be fixed upstream)